input/fsh/omrs-related-person.fsh

```diff
@@ -0,0 +1,137 @@
+Profile: OMRSRelatedPerson
+Parent: RelatedPerson
+Id: omrs-related-person
+Title: "OpenMRS Related Person"
+Description: "A FHIR RelatedPerson as understood by OpenMRS Relationship"
+* id 1..1
+* patient 1..1
+* identifier 1..1
+* address 0..*
+* name 0..*
+* gender 0..1
+* birthDate 0..1
+* active 1..1
+* period 1..1
+
+//OpenMRS RelatedPerson Example
+
+Instance: RelatedPersonExample
+InstanceOf: OMRSRelatedPerson
+Title: "Example of RelatedPerson"
+Description: "An example instance of the OpenMRS RelatedPerson resource."
+* patient = Reference(example-openmrs-Patient)
+* identifier.system = "http://openmrs.org/fhir/RelatedPerson"
+* identifier.value = "Person/1000Y"
+* name.id = "915c9146-8bdb-4fad-bc51-4af10d041c86"
+* name.family = "John"
+* name.given[0] = "Smith"
+* gender = #male
+* birthDate = "2001-09-08"
+* active = true
+* period.start = "2020-01-01"
+* period.end = "2024-01-01"
+
+Mapping: RelatedPersonMapping
```

Comment:
Use this [test example](https://github.com/openmrs/openmrs-module-fhir2/blob/b20caa03498f860807f90660c63a7a9566bf6fee/api/src/test/java/org/openmrs/module/fhir2/api/translators/impl/RelatedPersonTranslatorImplTest.java#L102) to get the actual mappings

Reply:
Updated, I removed active from mappings as it seems fsh doesn't support conditional logic, I was using "voided" as some other .fsh did this so I can add back the mapping if you think it should map to voided. I also removed the URL from the identifier system and added missing address. 

I wasn't sure if we should map the patient by reference or not, something like

`* patient.reference -> "Relationship.personB.uuid"`

I can change it to this if needed.

I didn't change much else for mappings so please let me know if I have missed anything